### DISCUSSION TITLE
第2节课作业(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#90] [作业]
13班同学们第二节课作业👇

**Comment by qingyin741** [comment]
+ 基础作业：
1.使用 InternLM-Chat-7B 模型生成 300 字的小故事
![Snipaste_2024-01-06_15-23-00](https://github.com/InternLM/tutorial/assets/118804949/08e5bbe5-f489-4997-83c1-49a662130867)
2.熟悉 hugging face 下载功能，使用 huggingface_hub python 包，下载 InternLM-20B 的 config.json 文件到本地
这个任务有很多问题:
1.直接下载，会报错，急得我抓耳挠腮，毫无办法，然后看了一下同学的操作：
修改下载源，下载指定文件
![Snipaste_2024-01-06_16-05-21](https://github.com/InternLM/tutorial/assets/118804949/d9b41162-bee3-4366-9bb7-9655e8136c66)
2.直接在本地下载没有使用开发机
![Snipaste_2024-01-06_17-40-58](https://github.com/InternLM/tutorial/assets/118804949/eec9d47b-97b2-4c06-b6d0-fe7e3c2bdc73)
![Snipaste_2024-01-06_17-42-26](https://github.com/InternLM/tutorial/assets/118804949/120e14be-0433-4054-89aa-a431f28ecfc4)


+ 进阶作业：

<img width="1376" alt="image" src="https://github.com/InternLM/tutorial/assets/118804949/edac6f33-0509-4555-b049-09eb0dbb6dfc">

第二个任务：浦语·灵笔的图文理解及创作部署 ，图片没生成出来哈哈
<img width="1239" alt="image" src="https://github.com/InternLM/tutorial/assets/118804949/4dabcd9c-5bcb-41ab-bc18-61fc7fe861a2">

**Comment by qingtao12138** [comment]
[浦语大模型实战课（hello_world）作业 (optimized).pdf](https://github.com/InternLM/tutorial/files/13849664/hello_world.optimized.pdf)

**Reply by qingtao12138** [reply]
完成基础和进阶作业～

**Reply by Lum1104** [reply]
同学可以建一个GitHub仓库或者写成csdn、知乎博客～

**Reply by qingtao12138** [reply]
好的嘞～！

**Comment by Tianhang-Lu** [comment]
https://zhuanlan.zhihu.com/p/676424555

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/TT3twAgaYi6NEMk8q51c0nRYnWg?from=from_copylink

**Comment by zhaoyiz0606** [comment]
下载 hugging face 中的文件
![1](https://github.com/InternLM/tutorial/assets/55607803/ffa12505-37a4-4fe7-8d42-3f854d9dc19c)
写一个小故事
![WX20240107-211626@2x](https://github.com/InternLM/tutorial/assets/55607803/22e3c1be-fddc-46a4-b4d1-510a21fe3ef4)

**Comment by qinb** [comment]
第二次作业 
https://docs.google.com/document/d/1RIKSrrGgb_9fI2uv-VffghW4ao0Cv3nbRtk1A_6etAY/edit?usp=sharing

**Comment by utopia2077** [comment]
作业2
https://nibqzjipp3p.feishu.cn/docx/Kcvxdrah9o9w08xAcBucHcnTnNc?from=from_copylink

**Comment by 5Fbw** [comment]
作业2
https://fifth-tower-f54.notion.site/46a51c678dd941b9b484eef2277b8f55?pvs=4

**Comment by bianchengyuanyuan** [comment]
作业 https://ou3v176g8k.feishu.cn/docx/NXW7dACXfoPoyPxydpscmRv7nMf

**Comment by SiliconSisyphus** [comment]
### 1.基础作业：
**（1）生成的小故事：**
第一次竟然失败了？其中回答显然字数不够
<img width="712" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/85b86e13-139e-4ef0-99fd-518387af00eb">
第二次字数够了
<img width="1051" alt="homework1" src="https://github.com/InternLM/tutorial/assets/152180631/ae8f8a0b-5135-4c4d-96c6-4aadfea4ef55">
**（2）熟悉 hugging face 下载功能：**
<img width="854" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/93d6e91f-4aa0-4389-80fa-f3dc714d613b">

###2.进阶作业
**（1）完成 Lagent 工具调用 Demo 创作部署：**
<img width="1225" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/1cd08e96-7d42-4dff-bd5b-9682119d2525">
**（2）完成浦语·灵笔的图文理解及创作部署：**
<img width="875" alt="image" src="https://github.com/InternLM/tutorial/assets/152180631/391691e2-846e-4408-a0bc-b505cea1e83c">

**Comment by Loewen-Hob** [comment]
http://t.csdnimg.cn/nkOhR
第二次作业

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135485280

**Comment by RessCris2** [comment]
【腾讯文档】第二次作业
https://docs.qq.com/doc/DREhOSFNwd21ac1F0

**Comment by suwief** [comment]
https://zhuanlan.zhihu.com/p/677639192
